index.html

```diff
@@ -56,6 +56,48 @@ <h2>Idea Title</h2>
             <h3>Comment</h3>
           </div>
         </div>
+        <div class="idea-box">
+          <div class="box-header">
+            <img src="assets/star-active.svg" alt="Star Icon" class="box-star">
+            <img src="assets/delete.svg" alt="Delete Icon" class="box-x">
+          </div>
+          <div>
+            <h2>Idea Title</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum suscipit consequat nibh, ac interdum dui ultricies id. Integer sit amet elementum felis. Sed pulvinar, risus in luctus aliquet, quam ex sagittis erat, sollicitudin egestas neque dui vel massa.</p>
+          </div>
+          <div class=" box-footer">
+            <img src="assets/comment.svg" alt="Add Comment Icon">
+            <h3>Comment</h3>
+          </div>
+        </div>
+        <div class="idea-box">
+          <div class="box-header">
+            <img src="assets/star-active.svg" alt="Star Icon" class="box-star">
+            <img src="assets/delete.svg" alt="Delete Icon" class="box-x">
+          </div>
+          <div>
+            <h2>Idea Title</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum suscipit consequat nibh, ac interdum dui ultricies id. Integer sit amet elementum felis. Sed pulvinar, risus in luctus aliquet, quam ex sagittis erat, sollicitudin egestas neque dui vel massa.</p>
+          </div>
+          <div class="box-footer">
+            <img src="assets/comment.svg" alt="Add Comment Icon">
+            <h3>Comment</h3>
+          </div>
+        </div>
+        <div class="idea-box">
+          <div class="box-header">
+            <img src="assets/star-active.svg" alt="Star Icon" class="box-star">
+            <img src="assets/delete.svg" alt="Delete Icon" class="box-x">
+          </div>
+          <div>
+            <h2>Idea Title</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum suscipit consequat nibh, ac interdum dui ultricies id. Integer sit amet elementum felis. Sed pulvinar, risus in luctus aliquet, quam ex sagittis erat, sollicitudin egestas neque dui vel massa.</p>
+          </div>
+          <div class="box-footer">
+            <img src="assets/comment.svg" alt="Add Comment Icon">
+            <h3>Comment</h3>
+          </div>
+        </div>
```

Comment:
This makes sense to copy/paste this so we can test functionality on desktop next.  Let's look into text limits for the <p> on 94, want to make sure we can deal with overflow when refactoring.